test/Analyzers/Microsoft.Extensions.LocalAnalyzers/ApiLifecycle/Data/Time.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "Name": "Time",
+  "Types": [
+    {
+      "Type": "sealed class Microsoft.R9.Extensions.Time.SystemClock2",
```

Comment:
is this still R9? can you search and replace "Microsoft.R9.Extensions"

Reply:
Done.